src/test/java/guitests/guihandles/BrowserPanelHandle.java

```diff
@@ -52,7 +52,13 @@ public void rememberUrl() {
      * {@code rememberUrl()} call.
      */
     public boolean isUrlChanged() {
-        return !lastRememberedUrl.equals(getLoadedUrl());
+        String googleUrl = "https://www.google.com.sg/maps/search/";
+        if (lastRememberedUrl.toString().substring(0, 38).equals(googleUrl)) {
+            return !lastRememberedUrl.toString().substring(0, 44)
+                    .equals(getLoadedUrl().toString().substring(0, 44));
+        } else {
+            return !lastRememberedUrl.equals(getLoadedUrl());
+        }
```

Comment:
why is this edited? oO

Reply:
Because google maps kept appending some weird characters at the end of the URL making it fail the comparison. By using the substring I can work around this.